scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -562,11 +562,21 @@ def load_from_file(self, file, filter_platform=[]):
             instance_list = []
             for ts in jtp.get("testsuites", []):
                 logger.debug(f"loading {ts['name']}...")
-                testsuite = ts["name"]
+                testsuite = os.path.normpath(ts["name"])
 
                 platform = self.get_platform(ts["platform"])
                 if filter_platform and platform.name not in filter_platform:
                     continue
+
+                if testsuite not in self.testsuites:
+                    # this could be a path issue in testplan and run environment
```

Comment:
please rephrase this comment, I do not understand what it means.

Reply:
the path issue is about we may have "tests//drivers//dma" which can not be convert to "test/driver/dma",  but per @gchwier 's recommadation, this is not required anymore, I will remove it